src/components/green-flag/green-flag.css

```diff
@@ -3,6 +3,8 @@
     height: 1.1rem;
     opacity: 0.5;
     margin: 0.25rem 0.6rem;
+    user-drag: none; 
```

Comment:
If the elements are still draggable, does the `user-drag` property give us anything?